src/libssh2_priv.h

```diff
@@ -642,6 +643,7 @@ struct _LIBSSH2_SESSION
 
     /* public key algorithms accepted as comma separated list */
     char *server_sign_algorithms;
+    size_t server_sign_algorithms_len;
```

Comment:
Is this used some place?

Reply:
No. It will be removed.

I still have `test_keyboard_interactive_auth_succeeds_with_correct_response` for `VS2015, WinCNG, Shared` failing.

There is no way for me to run tests on that platform without using CI so I think I will have to push a commit with crazy amounts of output to stderr to find out more about why it fails before having a chance to come up with a fix.

Reply:
NP, log away. :)